actions/ql/lib/codeql/actions/controlflow/internal/Cfg.qll

```diff
@@ -134,6 +134,10 @@ private module Implementation implements CfgShared::InputSig<Location> {
   SuccessorType getAMatchingSuccessorType(Completion c) { result = c.getAMatchingSuccessorType() }
 
   predicate isAbnormalExitType(SuccessorType t) { none() }
+
+  int idOfAstNode(AstNode node) { none() }
```

Comment:
Note that `getJoinBlockPredecessor` [isn't implemented](https://github.com/github/codeql/blob/17d2e4a3388a590dcb0513b0e4ada8ec371126eb/actions/ql/lib/codeql/actions/controlflow/BasicBlocks.qll#L323) in the Actions basic block library, so this instantiation is actually fine. And we could move Actions to the shared BB library in a future PR.